src/azure-cli/azure/cli/command_modules/serviceconnector/_help.py

```diff
@@ -270,6 +271,15 @@ def get_source_display_name(sourcename):
                 client-id      : Required. Client id of the user assigned identity.
                 subs-id        : Required. Subscription id of the user assigned identity.
         ''' if AUTH_TYPE.UserIdentity in auth_types else ''
+        workload_identity_param = '''
+            - name: --workload-identity
+              short-summary: The Microsoft Entra Workload ID with AKS.
+              long-summary: |
+                Usage: --workload-identity client-id=XX subs-id=XX
+
+                client-id      : Required. Client id of the workload identity, i.e., a user assigned identity.
+                subs-id        : Required. Subscription id of the workload identity, i.e., a user assigned identity.
```

Comment:
to make it simple, let's use a `--workload-identity <User-MI-Resource-ID>`

then, let's call `run_cli_cmd` to get the `client_id` and `subs_id` of the identity and pass it to client

param description: `An user-assigned managed identity used to create workload identity federation. (Please DO NOT use AKS control plane identity and kubelet identity which is not supported by federated identity credential.) `